e2e/Uno/HelloUnoWorld.Shared/Dialogs/AnotherContentDialog.xaml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Prism.Services.Dialogs;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace HelloUnoWorld.Dialogs
+{
+    /// <summary>
+    /// Interaction logic for AnotherDialogWindow.xaml
+    /// </summary>
+    public partial class AnotherContentDialog : ContentDialog, IDialogWindow
+    {
+        public AnotherContentDialog()
+        {
+            InitializeComponent();
+        }
+
+        public IDialogResult Result { get; set; }
+
+        event RoutedEventHandler IDialogWindow.Loaded
+        {
+            add => Loaded += value;
+            remove => Loaded -= value;
+        }
+
+        event TypedEventHandler<ContentDialog, ContentDialogClosingEventArgs> IDialogWindow.Closing
+        {
+            add => Closing += value;
+            remove => Closing -= value;
+        }
+
+        event TypedEventHandler<ContentDialog, ContentDialogClosedEventArgs> IDialogWindow.Closed
+        {
+            add => Closed += value;
+            remove => Closed -= value;
+        }
+    }
+}
```

Comment:
We really should be using the DialogWindow automagically from the dialog service. it's what should be used when a DialogWindow is not specified. The Dialog should be any acceptable View Type and then added as the child of the Dialog Window.

Reply:
That part is not used in the e2e scenario, so it could be either a sample, or we can remove entirely.

Reply:
we can always build out more functionality into the e2e app over time... so if we're not using it or have a way to manually test it right now it's fine to remove for this scenario

Reply:
I've removed that sample.